Modules/System/XML Validation/src/XmlValidation.Codeunit.al

```diff
@@ -0,0 +1,55 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Provides helper functions for xml validation against a schema.
+/// </summary>
+
+codeunit 50100 "Xml Validation"
+{
+    Access = Public;
+
+    var
+        XmlValidationImpl: Codeunit "Xml Validation Impl.";
+
+    /// <summary>
+    /// Performs validation of an xml from a string against a schema from a string.
+    /// </summary>
+    /// <param name="Xml">Xml string to validate.</param>
+    /// <param name="XmlSchema">Xml schema string to validate against.</param>
+    /// <param name="Namespace">Namespace of the xml schema.</param>
+    /// <param name="ErrorText">The error text if the validation was not successful.</param>
+    /// <returns>True if validation was successful, false otherwise.</returns>
+    procedure ValidateAgainstSchema(Xml: Text; XmlSchema: Text; Namespace: Text; var ErrorText: Text): Boolean
```

Comment:
Instead of passing `ErrorText`, you can simply make the function a try function. Then the consumer can decide what to do with the error.

Reply:
Changed the ValidateAgainstSchema function into TryFunction and removed ErrorText parameter.